src/User/UserServiceProvider.php

```diff
@@ -84,6 +98,15 @@ public function boot()
             User::addGroupProcessor($callback);
         }
 
+        $passwordCheckers = array_map(function ($checker) {
+            if (is_string($checker)) {
+                $checker = $this->app->make($checker);
```

Comment:
I'm concerned that this gets resolved every time Flarum boots, even though password validation is something that happens very rarely during the lifetime of the app. I'd advocate a solution similar to https://github.com/flarum/core/pull/2423#discussion_r517028321

Reply:
I'll update this when #2423 is merged